landing-page/scripts/tests/test_encode_to_mei.py

```
@@ -528,3 +528,95 @@ def test_verify_and_correct_syllables_applies_a_genuine_text_disagreement():
+            _box("Alleluia", 5, 95, 190, 210),   # genuinely s1's own text
+            _box("Kyrie", 5, 95, 305, 325),        # genuinely s2's own text
+        ]
+    }
+
+    xml_bytes = mei.build_mei(
+        glyphs_by_stave, staves,
+        image_path=Path("page.jpg"), image_w=2000, image_h=2000,
+        manuscript_name="test", text_alignment=text_alignment,
+        n_detected_staves=2,
+    )
+    root_before = ET.fromstring(xml_bytes)
```

Ruff (0.16.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)